hub/ui/src/components/filter/Filter.tsx

```diff
@@ -76,7 +76,7 @@ const Filter: React.FC = (props: any) => {
     fetch(`${API_URL}/categories`)
       .then((res) => res.json())
       .then((categoryData) =>
-        setCategoriesList(addCategory(categoryData)));
+        setCategoriesList(addCategory(categoryData.data)));
```

Comment:
@PuneetPunamiya @SM43 shouldn't this have resulted in some tests changing as well? 
I wonder if this is pointing to lack of critical tests.